src/Our.Umbraco.AuthU/Data/Plans/RegisterOAuthTables.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Our.Umbraco.AuthU.Data.Helpers;
+using Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthClientStore;
+using Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthRefreshTokenStore;
+using Our.Umbraco.AuthU.Models;
+using System;
+using System.Linq;
+using Umbraco.Core.Composing;
+using Umbraco.Core.Migrations;
+
+namespace Our.Umbraco.AuthU.Data.Migrations.Plans
+{
+    /// <summary>
+    /// This is the plan that runs the AuthU Migrations
+    /// </summary>
+    internal class RegisterOAuthTables : MigrationPlan
+    {
+        /// <summary>
+        /// Add the demo id if preferred
+        /// </summary>
+        /// <param name="CreateDemoClient"></param>
+        public RegisterOAuthTables(bool CreateDemoClient = false) : base("AuthU")
+        {
+
+            if (!CreateDemoClient)
+            {
+                From(InitialState)
```

Comment:
Duplication here, but given the above comment, I think we only need one route anyway.